packages/FormElement/src/hooks/useFormElement.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import nanoid from "nanoid";
+import isNil from "lodash/isNil";
+
+export default function useFormElement(props = { id: "", wrapperA11yProps: {} }) {
+  const { id, wrapperA11yProps } = props;
+  const wrapperAriaDescribedBy = wrapperA11yProps ? wrapperA11yProps["aria-describedby"] : "";
+  const ariaDescriptionId = React.useRef(nanoid()).current;
```

Comment:
state is better for nanoid https://github.com/ai/nanoid#react

Reply:
Why? It doesn't change or need to invoke a rerender, so why not just use a static ref?

Reply:
pretty sure it's a sub-millisecond matter but with a ref there's a `nanoid()` call happening on every invocation of the outer function even though its result is immediately discarded. not so with state